app/scripts/models/reliers/oauth.js

```diff
@@ -159,7 +159,13 @@ define([
           //jshint camelcase: false
           // server version always takes precedent over the search parameter
           self.set('redirectUri', serviceInfo.redirect_uri);
+          self.set('termsUri', serviceInfo.terms_uri);
+          self.set('privacyUri', serviceInfo.privacy_uri);
+          self.set('trusted', serviceInfo.trusted);
```

Comment:
Do we know all reliers would have **_both_** a terms of service and privacy policy?
All the sites i randomly checked seemed to have them, although it looks like Atlassian only has a single page, ["Legal Terms"](https://www.atlassian.com/end-user-agreement/).